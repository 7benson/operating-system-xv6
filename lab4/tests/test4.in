# x a
4 4
1,2,8,2, 
3,6,3,4, 
9,2,12,2, 
3,17,3,4, 

3 1
1,
2,
3,