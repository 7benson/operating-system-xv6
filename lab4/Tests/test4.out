3 1
2377 
2378 
2379 
